### ISSUE TITLE
Hindenburg not working anymore (Among us Update)

### ISSUE THREAD

**Issue by koxx12-dev** [SkeldJS/Hindenburg#58]
Making this issue because among us just now updated and it broke Hindenburg 

log
```
error: Unidentified (1, 2ms) sent a malformed packet
hindenburg~$ RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 40. Received 41
    at new NodeError (node:internal/errors:371:5)
    at boundsError (node:internal/buffer:86:9)
    at Uint8Array.readUInt8 (node:internal/buffer:252:5)
    at HazelReader.uint8 (Hindenburg\node_modules\@skeldjs\util\lib\HazelReader.ts:89:34)
    at HazelReader.packed (Hindenburg\node_modules\@skeldjs\util\lib\HazelReader.ts:330:31)
    at Function.Deserialize (Hindenburg\src\packets\ModdedHelloPacket.ts:59:37)
    at PacketDecoder._parse (Hindenburg\node_modules\@skeldjs\protocol\lib\PacketDecoder.ts:544:44)
    at PacketDecoder._parse (Hindenburg\node_modules\@skeldjs\protocol\lib\PacketDecoder.ts:535:25)
    at PacketDecoder.parse (Hindenburg\node_modules\@skeldjs\protocol\lib\PacketDecoder.ts:583:21)
    at Worker.handleMessage (Hindenburg\src\Worker.ts:1537:47) {
  code: 'ERR_OUT_OF_RANGE'
}
```

###### i prob should have made this issue in skieldjs repo but eh

**Comment by edqx** [comment]
Thanks, I'll get this fixed asap.

This is fine in the Hindenburg repo.

**Comment by edqx** [comment]
Fixed in f3c94939a53e4fc502894cfdcec9eb7e459773e0 (1.0.7)
